packages/ui/src/Field.tsx

```diff
@@ -36,58 +30,46 @@ export interface FieldProps extends FieldWithLabelsProps {
     | "address"
     | "customSelect";
   rows?: number;
+  value?: any;
+  setValue?: any;
   options?: SelectListOptions;
   placeholder?: string;
   disabled?: boolean;
 }
 
 /**
- * Field is a fully uncontrolled component for creating various inputs. Fully uncontrolled means Field manages its own
- * state for the TextFields/Switches/etc, not the parent component. When values are updated, Field will pass the data to
- * the parent through the handleChange prop. You can set an initialValue, but you should not update initialValue
- * this prop with the result of handleChange.
- *
  * Note: You MUST set a key={} prop for this component, otherwise you may wind up with stale data. Just changing
  * initialValue will not work.
  *
  */
-export function Field(props: FieldProps) {
-  const [value, setValue] = useState(props.initialValue || "");
-  const [multiselectValue, setMultiselectValue] = useState(props.initialValue ?? []);
-
+export const Field = ({
+  name,
+  label,
+  labelColor,
+  height,
+  type,
+  rows,
+  value,
+  setValue,
```

Comment:
I'd suggest something like onChange etc to match most other components in RN.

Reply:
Seems reasonable.